### PR TITLE
filter table with master controller

### PR DESCRIPTION
**Pull request by Nikam1999123**

### CODE REVIEW

File: src/main/java/CLSACloneMicronaut/model/ClientTiers.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+@Entity
+@Table(name = "client_tiers")
+public class ClientTiers {
+    @Id
+    @Column(name = "clienttiers_id", nullable = false, unique = true)
+    private String clienttiersId;
+
+    @NotEmpty
+    @Column(name = "clienttiers_name", nullable = false)
+    private String clienttiersName;
```
Comment (akshay5147) [inline src/main/java/CLSACloneMicronaut/model/ClientTiers.java:15]: put varuiable name - clienttiersId in camel case

File: src/main/java/CLSACloneMicronaut/model/ClientTiers.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+@Entity
+@Table(name = "client_tiers")
+public class ClientTiers {
+    @Id
+    @Column(name = "clienttiers_id", nullable = false, unique = true)
+    private String clienttiersId;
+
+    @NotEmpty
+    @Column(name = "clienttiers_name", nullable = false)
```
Comment (akshay5147) [inline src/main/java/CLSACloneMicronaut/model/ClientTiers.java:14]: clienttiers_name - check the naming conventions it should be like client_tier_name or tier_name
same for other column names fields

File: src/main/java/CLSACloneMicronaut/model/ExecutionVenue.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
+package CLSACloneMicronaut.model;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "ExecutionVenue")
```
Comment (akshay5147) [inline src/main/java/CLSACloneMicronaut/model/ExecutionVenue.java:8]: keep similar naming convention for table name

File: src/main/java/CLSACloneMicronaut/model/ExecutionVenue.java [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    @Id
+    @Column(name = "venue_id", nullable = false, unique = true)
+    private String venueId;
+
+    @NotEmpty
+    @Column(name = "venue_name", nullable = false)
+    private String venueName;
+
+    @NotNull
+    @Column(name = "MIC_Code", nullable = false)
```
Comment (akshay5147) [inline src/main/java/CLSACloneMicronaut/model/ExecutionVenue.java:19]: all letters should be in a small case

File: src/main/java/CLSACloneMicronaut/model/ClientTiers.java (hunk 2 above)
Comment (Nikam1999123) [reply src/main/java/CLSACloneMicronaut/model/ClientTiers.java]: t have set like  client_tier_name

File: src/main/java/CLSACloneMicronaut/model/ExecutionVenue.java (hunk 4 above)
Comment (Nikam1999123) [reply src/main/java/CLSACloneMicronaut/model/ExecutionVenue.java]: update mic_code

**Pull request merged**